s2core/src/main/scala/org/apache/s2graph/core/S2Graph.scala

```diff
@@ -282,12 +284,12 @@ class S2Graph(_config: Config)(implicit val ec: ExecutionContext) extends S2Grap
   *  right implementation(S2GRAPH-213).
   * */
   override def getVertexFetcher(column: ServiceColumn): VertexFetcher = {
-    resourceManager.getOrElseUpdateVertexFetcher(column)
+    resourceManager.getOrElseUpdateVertexFetcher(column, Option(60 * 60 * 24))
```

Comment:
Is it possible for us to use system wise default from config(such as "resource.cache.ttl.sseconds") instead of constant?

Reply:
I agree with what you set as the "resource.cache.ttl.sseconds" value.